src/main/java/mods/eln/node/NodeBase.java

```diff
@@ -153,6 +153,10 @@ public NodeBase() {
 
     boolean destructed = false;
```

Comment:
I'm a bit leery of this sort of patch, so let me think out loud.

It'll *work*, no doubt about that, but only so long as a bunch of special cases are caught. What's actually going on is... the node is getting reconnected after it's been supposedly destroyed, when the UI closes. A symptom of the simulation existing in a parallel world from Minecraft, I suppose; the block is gone, but the node is not.

Ideally we'd tie them a bit more closely together. How often is connectJob called? Often enough... checking for the block's existence every time it's run might degrade performance, or have other problems.

Maybe we could add a process that, at a relatively slow pace, checks each node to make sure it still exists in-world? That would also catch other causes of this sort of thing. It should cry havoc to the console if it finds anything, of course, but it's a fallback and debugging mechanism.